### PR TITLE
week 1. Modules  1-12

### PR DESCRIPTION
**Pull request by lis-anna**

### CODE REVIEW

File: 01-node-tutorial/answers/content/01-intro.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const amount = 12;
```
Comment (aschey) [inline 01-node-tutorial/answers/content/01-intro.js]: I think this is supposed to be in the answers folder

File: 01-node-tutorial/answers/09-path-module [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const path = require("path");
```
Comment (aschey) [inline 01-node-tutorial/answers/09-path-module]: Looks like this file is missing the `.js` suffix

**Comment by lis-anna** [comment]
I changed the file name and location according to your notes @aschey

**Review by aschey — APPROVED** [review]

**Comment by aschey** [comment]
Thanks, feel free to go ahead and merge

**Pull request merged**